portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,68 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Displays random fact about me
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  var num = 100;
+  const facts =
+      ['I am 20 years old', 'I live in Colorado', 'I am a human', 'I am not a robot'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factsContainer = document.getElementById('fact');
+  factsContainer.innerText = fact;
+
+  // Adds function to move text back and forth
+  if (factsContainer.style.marginLeft == "")
+    setInterval( function() {
+            factsContainer.style.marginLeft = (Math.abs(100 - num)%100).toString() + "px";
+            num = num % 199 + 1;
+            console.log((Math.abs(100 - num)%100).toString() + "px");
+        }, 20);
+}
+
+function hideUnhide(content)  {
+    if (content.style.display === "block") {
+            content.style.display = "none";
+            } else {
+            content.style.display = "block";
+            }
+}
+
+function collabsible() {
+    var coll = document.getElementsByClassName("collapsible");
+        var i;
+
+        for (i = 0; i < coll.length; i++) {
+        coll[i].addEventListener("click", function() {
+            this.classList.toggle("active");
+            var content = this.nextElementSibling;
+            if (content.style.display === "block") {
+            content.style.display = "none";
+            } else {
+            content.style.display = "block";
+            }
+        });
+        }
+}
+
+function dispM1() {
```

Comment:
Same as above, let's use a hidden class. Could you make these function names clearer? toggleMotorcycleImageVisibility() eg.

Reply:
Uses hidden class and has better name